src/components/blocks/object-shuffler/_object-shuffler.scss

```diff
@@ -97,18 +97,17 @@
   }
 
   &__slide {
-    align-content: space-between;
     display: flex;
     flex-flow: row wrap;
     height: 100%;
-    justify-content: space-between;
     left: 0;
     margin: 0 3% 0 7%;
+    place-content: space-between;
     position: absolute;
     top: 0;
     width: 90%;
 
-    @media (min-width: 1000px) {
+    @media (width >= 1000px) {
```

Comment:
i think this syntax is much clearer!
but why not make the change everywhere?

Reply:
@methodog Well, interestingly this highlights an issue with the linting in that errors are reported when the media feature range is in the following format

`@media (min-width: 1000px) {...}`

but if we are using calculations and/or variables then this is _not_ reported:

`@media (min-width: map-get($nav-breakpoints, $breakpoint)) {...}`

So it seems we should keep in mind when addressing linting errors and indeed developing from new that there may be other somewhat hidden occurrences. If a search across the vam-fractal project is made using `@media (min-` then other instances can be found in the following files:

src/components/blocks/site-nav/_site-nav.scss
src/components/mixins/breakpoints/readme.md
src/components/mixins/breakpoints/_breakpoints.scss

Furthermore, a search for `@media (max-` reveals further instances not detected by the linter in:

 src/components/blocks/site-nav/_site-nav.scss
src/components/mixins/breakpoints/_breakpoints.scss

Reply:
thanks Nick. 
interesting point about the syntax change being entirely a SASS abstraction - hopefully CSS will catch up with the improved syntax!
back on the point about switching to the new syntax consistently throughout, presumably we are still free to do that, even if not forced to by the linter?

Reply:
ah - it looks as though you did just that, with a commit between my comments.